module/apmgormv2/go.mod

```diff
@@ -1,13 +1,16 @@
 module go.elastic.co/apm/module/apmgormv2/v2
 
 require (
-	github.com/stretchr/testify v1.6.1
+	github.com/mattn/go-sqlite3 v1.14.16 // indirect
+	github.com/stretchr/testify v1.8.0
 	go.elastic.co/apm/module/apmsql/v2 v2.2.0
 	go.elastic.co/apm/v2 v2.2.0
+	golang.org/x/crypto v0.3.0 // indirect
 	gorm.io/driver/mysql v1.0.2
-	gorm.io/driver/postgres v1.0.2
-	gorm.io/driver/sqlite v1.1.4-0.20200928065301-698e250a3b0d
-	gorm.io/gorm v1.20.2
+	gorm.io/driver/postgres v1.4.5
+	gorm.io/driver/sqlite v1.4.3
+	gorm.io/driver/sqlserver v1.4.1
+	gorm.io/gorm v1.24.2
```

Comment:
The dependency version bumps have broken Go 1.15 support, so we'll need to revert these.

Reply:
See latest [commit](https://github.com/elastic/apm-agent-go/pull/1356/commits/a4a2db9082769a1b4a48bb1166aba12b5c3ee39b). 

Ran `make check` inside a container running golang v1.15.15 so should be okay. We'll see with CI tests.

Note: The older versions of the `gorm.io/driver/sqlserver` package use the driver `github.com/denisenkom/go-mssqldb` and were only recently updated to use `github.com/microsoft/go-mssqldb`, hence the updates surrounding that.